src/components/input-field/input-field.scss

```diff
@@ -10,6 +10,7 @@
 
 @import './partial-styles/trailing-icon.scss';
 @import './partial-styles/outlined-style-overrides.scss';
+@import './partial-styles/cropped-label-hack.scss';
```

Comment:
Should we use the underscore prefix since this is a partial?

Reply:
we should. iForgot
Will you please fix it up? Or you want me to do so?

Reply:
I can do it 👍